app/views/admin/users/_capability_checkbox.html.erb

```diff
@@ -1,3 +1,4 @@
+<%= 'SKIP' if capability == 'skip' %>
```

Comment:
Consider replacing the if/unless on lines 1 & 2 with an if/else.

Reply:
Huh. That line was debug output ... that never worked and hence didn't get removed! :D